mesheryctl/internal/cli/root/perf/perf.go

```diff
@@ -32,6 +34,11 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
+	mesheryAuthToken = "http://localhost:9081/api/gettoken"
+	mesheryURL       = "http://localhost:9081/api/load-test-smps?"
```

Comment:
_A note to be filed as a separate issue:_ These two constants should be global variables (or similar). We'll want to provide a `mesheryctl system config` to allow the user to update the location of Meshery. Meshery will not always be running on `localhost`.

Reply:
Hmm do we want to move it to a config file in that case? Global variables might lead to problems later on ?1

Reply:
Or a persistent flag 🤔

Reply:
A `mesheryctl.yaml` (or similar) config file sounds like a better approach, yes.

A persistent flag in combination with a config file also sounds nice. This would provide a flag that could be used when scripting `perf` tests for one-off executions against different environments. If a persistent flag is brought forward for overriding client defaults, we'll need to exercise diligence in its naming vs. the current global flag `--config`:

```
Global Flags:
      --config string   config file (default location is: /Users/lee/.meshery/meshery.yaml)
```

Reply:
`.meshery/config/config.yaml` which holds just these two entries for now and also add persistent flag to override. Correct?

Reply:
@leecalcote Let's move this to another issue?

Reply:
Yes, regarding the persistent flag for override.

Good suggestion on the name and location of the client config file. In this context, the name of the `/config` directory is a little misleading in that is retains a local database (bitcask) used for caching user preferences (user **config**uration information) and performance test results.

The `.meshery/meshery.yaml` currently serves as the Meshery Server "configuration file" (in quotes because, while users can configure server deployment, this file is a docker-compose file, so control over server behavior through this file format is somewhat limited). 

Perhaps, `.meshery/mesheryctl.yaml` could serve as the Meshery Client configuration file, which places it next to the server configuration file, but differentiations with a "ctl" on the end.

Reply:
> @leecalcote Let's move this to another issue?

Sounds real good. 👍

Reply:
Here we are - https://github.com/layer5io/meshery/issues/1022